jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -215,6 +215,10 @@ public void apply(Project project) {
                   .getSourceSets()
                   .getByName(SourceSet.MAIN_SOURCE_SET_NAME);
           jibDependencies.add(mainSourceSet.getRuntimeClasspath());
+          jibDependencies.add(
+              projectAfterEvaluation
```

Comment:
Quick question, as I don't have a lot of Gradle expertise: would it be better if line 212 used `projectAfterEvaluation` instead of `project` (e.g., at least for consistency and eliminating future questions like this)?

Reply:
From a technical perspective ist irrelevant, the `Project` instance changes while evaluation